tests/t.py

```diff
@@ -30,6 +30,7 @@
     #     'protocol': 'https',
```

Comment:
There are lots of commented blocks in this whole file, I expect it must be on purpose but just wanted to add a reminder to double check.